en/docs/how-tos/rate-limiting/distributed-throttling.md

```diff
@@ -1,14 +1,30 @@
-
-The API Microgateway upon recieving a request, checks against the local counter and if throttling limit  has not exceeded it publishes the events via a stream to a central traffic management solution. This is done over HTTP. The  central traffic management solution then  executes throttle policies against the events streams. When a particular request is throttled, the  central traffic management solution sends the details of the throttled out event to a JMS topic. Each API Microgateway node is subscribed to this JMS topic, and updates the local counter when the JMS topic is updated.  Hence the API Microgateway nodes gets notified of the throttle decisions through JMS messages.
-
-![distributed throttling]({{base_path}}/assets/img/how-tos/distributed-throttling.png)
+WSO2 API Microgateway has an in-memory mechanism by default, to handle throttling
+([node-level throttling](/how-tos/rate-limiting/adding-throttling-policies)).  
+In a deployment with multiple microgateways, throttling becomes a challenge with node local throttling as the throttling 
+decision is made based on the local counter within each node. If we proceed with the node local throttling in such 
+environment, the API user would be allowed to send multiples of the throttling limit.I.e. if the throttling limit is set to 10, 
+if we have 3 gateways in a cluster, it will allow 30 requests to pass to the backend, before all three gateways 
+throttle out requests. This will put an unexpected load on the backend. To address this requirement, the API Microgateway 
+supports distributed throttling where it is able to work with a central traffic management solution. In this case, 
+multiple microgateways can connect with WSO2 API Manager 
+([WSO2 Traffic Manager](https://apim.docs.wso2.com/en/latest/install-and-setup/setup/distributed-deployment/product-profiles/)) 
+and perform rate-limiting precisely.
+
+!!! note
+    The WSO2 Traffic Manager refers to the WSO2 API Manager running in 
+    [Traffic Manager Profile](https://apim.docs.wso2.com/en/latest/install-and-setup/setup/distributed-deployment/product-profiles/). 
+    If you start the WSO2 API Manager without providing any profile, it runs as All in One Node (All the profiles 
+    are activated). For testing purposes, you can simply start the API Manager following quick start guide and test 
```

Comment:
"If you start the WSO2 API Manager without providing any profile, it runs as All in One Node (All the profiles are activated). For testing purposes, you can simply start the API Manager following quick start guide and test" I suppose we can change the sentence/ change the order to make this less confusing?

Reply:
yes. I will make the change.

Reply:
updated via https://github.com/wso2/docs-mg/pull/41/commits/dae05bde11563c05e47375d5d63004a64b7ea4e4